### ISSUE TITLE
Erizo Controller memory leak

### ISSUE THREAD

**Issue by mkhahani** [lynckia/licode#1085]
![ec-memory-leak](https://user-images.githubusercontent.com/859377/33697790-ffb8ef66-db1d-11e7-8f99-0b6c2fbaef71.png)

As the image describes there is a noticeable increase in the memory usage by EC after 22 days running.

**Comment by jcague** [comment]
yes, that's definitively something that we should review, thanks for the heads up!
